src/test/c_api_tests.cpp

```diff
@@ -41,14 +44,54 @@ static void testDefaultSingleModelOptions(MultiModelOptionsImpl* mmo) {
     EXPECT_EQ(mmo->idleSequenceCleanup, std::nullopt);
 }
 
+#define ASSERT_CAPI_STATUS_NULL(code)         \
+    {                                         \
+        auto* err = code;                     \
+        if (err != nullptr) {                 \
+            const char* msg = nullptr;        \
+            OVMS_StatusGetDetails(err, &msg); \
+            std::string smsg(msg);            \
+            OVMS_StatusDelete(err);           \
+            ASSERT_TRUE(false) << smsg;       \
+        }                                     \
+    }
+
+#define ASSERT_CAPI_STATUS_NOT_NULL(code) \
+    {                                     \
+        auto* err = code;                 \
+        if (err != nullptr) {             \
+            OVMS_StatusDelete(err);       \
+        } else {                          \
+            ASSERT_TRUE(false);           \
```

Comment:
```suggestion
            ASSERT_NE(err, nullptr);           \
```

Reply:
This assert will never work, at this point err is always nullptr

Reply:
done